tube-player/modules/08-Add-API-endpoints/README.md

```diff
@@ -17,15 +17,20 @@ To interact with the remote endpoints you will utilize the Uno Platform HTTP ext
 
 To learn more about these extensions, refer to the [HTTP extension docs](xref:Overview.Http).
 
-### Install NuGet packages
+### Configure Uno Platform Features
 
-Open the NuGet package manager for the *TubePlayer* project and install the following packages:
+Open the TubePlayer.csproj file.
 
-- [`Uno.Extensions.Http.Refit`](https://www.nuget.org/packages?q=Uno.Extensions.Http.Refit)
-- [`Uno.Extensions.Http.WinUI`](https://www.nuget.org/packages?q=Uno.Extensions.Http.WinUI)
+Locate the <UnoFeatures> property within the <PropertyGroup> section.
```

Comment:
This text isn't displayed as expected.

![image](https://github.com/unoplatform/workshops/assets/114886335/983161d1-44e3-413d-9a4b-c5f5c954b281)